src/keymap.cpp

```diff
@@ -139,7 +139,7 @@ static op_desc opdescs[] = {
 	},
 	{
 		OP_SETTAG, "set-tag", "t",
-		_("Select tag"),
+		_("Set tag"),
```

Comment:
`set-tag` is an alias for `select-tag`, so there's no reason to document it differently. Can you please remove this change?

I don't know why the existence of this alias was necessary in the first place, but now we're stuck with it until we bump to 3.0.